optuna/samplers/_cmaes.py

```diff
@@ -58,7 +58,8 @@ def objective(trial):
 
         x0:
             A dictionary of an initial parameter values for CMA-ES. By default, the mean of ``low``
-            and ``high`` for each distribution is used.
+            and ``high`` for each distribution is used. Note that x0 is sampled uniformly within
+            the search space domain for each restart if you set ``restart_strategy`` argument.
```

Comment:
memo: I comment the reason why I choose this design here.

In pycma's implementation, `x0` accepts not only the float values. It also accepts callable object (e.g. `np.random.rand`) and string (e.g. `"np.random.rand"` which is evaluated by `eval()`). But I guess it's enough for almost every usecases to sample `x0` uniformly at each restart.